src/telex/telex-connection.controller.ts

```diff
@@ -21,8 +21,10 @@ export class TelexConnectionController {
 
   @Get()
   @ApiOkResponse({ description: 'All active TELEX connections', type: [TelexConnection] })
```

Comment:
The response example still shows the array.
![image](https://user-images.githubusercontent.com/1652722/98866939-055e4a00-246e-11eb-9ae5-96fb50f5a784.png)

Need to change the Paginated<> from an interface to a class. This way `@ApiProperty` can be added and the `@ApiOkResponse` type be set to `Paginated`. Generic types are not supported, unfortunately. This is the closest we can get.